api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -225,7 +226,7 @@ private JSONObject dataUriToJson(String dataUri) {
   public void testListClusters() throws Exception {
     when(notebookService.getCluster(WORKSPACE_NS, "all-of-us")).thenReturn(testFcCluster);
 
-    assertThat(clusterController.listClusters().getBody().getDefaultCluster())
+    assertThat(clusterController.listClusters(WORKSPACE_NS).getBody().getDefaultCluster())
```

Comment:
Same as above: consider renaming this to CLOUD_PROJECT_ID or whatever we decide on as the canonical term for billing project / GCP project IDs.